sense-api/gradle.properties

```diff
@@ -1 +1 @@
-release_version=0.0.1
\ No newline at end of file
+release_version=1.0.0
```

Comment:
Please, do not update the major version for now. Let's update the maintenance. The sense was not tested in production yet